### ISSUE TITLE
Please use cocoapod resource_bundles for PrivacyInfo

### ISSUE THREAD

**Issue by RayJiang16** [devicekit/DeviceKit#396]
like sentry https://github.com/getsentry/sentry-cocoa/pull/3651
otherwise we cannot build project

**Comment by baiyidjp** [comment]
How did you adapt this library?  @RayJiang16

**Comment by RayJiang16** [comment]
I added PrivacyInfo.xcprivacy to the main project.
This is a demo that cannot be build.
[TestPrivacyInfo.zip](https://github.com/devicekit/DeviceKit/files/14661523/TestPrivacyInfo.zip)

**Comment by li-jun** [comment]
please use 
```
s.resource_bundles = {"DeviceKit" => ["Source/PrivacyInfo.xcprivacy"]}
```
instead of
```
s.resources                  = 'Source/PrivacyInfo.xcprivacy'
```

Otherwise, the PrivacyInfo file in the main project will be overwritten.

**Comment by anton-plebanovich** [comment]
Resolved in https://github.com/devicekit/DeviceKit/pull/397
I guess we may use fork repo and branch as a workaround for now

**Comment by Zandor300** [comment]
Thanks @RayJiang16 for your fix. Merged and released just now in v5.2.3!
